docs/user_manual/source/rvfi.rst

```diff
@@ -240,10 +240,41 @@ Instructions will populate the ``rvfi_mem`` outputs with incrementing ``NMEM``,
+``rvfi_mem_memtype`` indicates the memory type attributes associated with each memory operation (i.e cacheable or bufferable). For misaligned transactions that are
+split in two memory operations ``rvfi_mem_memtype`` will only report the type attribute for the first memory operation.
 ``rvfi_mem_atop`` indicates the type of atomic transaction as specified in [OPENHW-OBI]_.
-``rvfi_mem_exokay``  indicates the status of ``data_exokay_i`` for loads, non-bufferable stores and atomic instructions.
-``rvfi_mem_err`` indicates if a load, non-bufferable store or atomic instruction got a bus error.
+``rvfi_mem_exokay``  indicates the status of ``data_exokay_i`` for loads, non-bufferable stores and atomic instructions. For split transactions, ``rvfi_mem_exokay`` will only
+be 1 if both transactions receive ``data_exokay_i == 1``.
+``rvfi_mem_err`` indicates if a load, non-bufferable store or atomic instruction got a bus error. :numref:`rvfi_mem_err encoding for different transaction types` shows how
+different memory transactions report ``rvfi_mem_err``.
+
+.. table:: rvfi_mem_err encoding for different transaction types
+  :name: rvfi_mem_err encoding for different transaction types
+  :widths: 60 5 5 5 60
+  :class: no-scrollbar-table
+
+  +---------------------+---------+----------------+----------------+--------------------------------+
+  | Instruction type    | Split   | Bufferable (1) | Bufferable (2) |  rvfi_mem_err                  |
+  +=====================+=========+================+================+================================+
+  | Load                | No      | N/A            | N/A            | data_err_i                     |
+  +---------------------+---------+----------------+----------------+--------------------------------+
+  | Load                | Yes     | N/A            | N/A            | data_err_i(1) OR data_err_i(2) |
+  +---------------------+---------+----------------+----------------+--------------------------------+
+  | Store               | No      | No             | N/A            | data_err_i                     |
+  +---------------------+---------+----------------+----------------+--------------------------------+
+  | Store               | No      | Yes            | N/A            | 0                              |
+  +---------------------+---------+----------------+----------------+--------------------------------+
+  | Store               | Yes     | No             | No             | data_err_i(1) OR data_err_i(2) |
+  +---------------------+---------+----------------+----------------+--------------------------------+
+  | Store               | Yes     | Yes            | Yes            | 0                              |
+  +---------------------+---------+----------------+----------------+--------------------------------+
+  | Store               | Yes     | Yes            | No             | data_err_i(2)                  |
```

Comment:
As mentioned internally, in this case rvfi_mem_type is registrated as bufferable, and there is nothing indicating that the second obi transfer is non-bufferable. You could therefor not know if the rvfi_mem_err and rvfi_mem_exokay is "correct" or just tied off.

As we discussed this will be fixed later, but wanted to mention the seemingly problematic scenario.